lib/db2.js

```diff
@@ -76,6 +76,8 @@ DB2.prototype.create = function(model, data, options, callback) {
  * @param {Function} cb The callback function
  */
 DB2.prototype.update = function(model, where, data, options, cb) {
+  debug('DB2.prototype.update (enter): where=%j, data=%j, options=%j\n',
```

Comment:
I would prefer to have this change in a separate commit with appropriate commit description; it is out of the scope of implementing `replaceById`